python/core/qgsstatisticalsummary.sip

```diff
@@ -1,214 +1,299 @@
-     * @see addVariant()
-     * @see finalize()
-     * @note added in QGIS 2.16
-     */
     void addValue( double value );
+%Docstring
+ Adds a single value to the statistics calculation. Calling this method
+ allows values to be added to the calculation one at a time. For large
+ quantities of values this may be more efficient then first adding all the
+ values to a list and calling calculate().
+ \param value value to add
+.. note::
+
+   call reset() before adding the first value using this method
+ to clear the results from any previous calculations
+.. note::
+
+   finalize() must be called after adding the final value and before
+ retrieving calculated statistics.
+.. seealso:: calculate()
+.. seealso:: addVariant()
+.. seealso:: finalize()
+.. versionadded:: 2.16
+%End
+
+    void addVariant( const QVariant &value );
+%Docstring
+ Adds a single value to the statistics calculation. Calling this method
+ allows values to be added to the calculation one at a time. For large
+ quantities of values this may be more efficient then first adding all the
+ values to a list and calling calculate().
+ \param value variant containing to add. Non-numeric values are treated as null.
+.. note::
+
+   call reset() before adding the first value using this method
+ to clear the results from any previous calculations
+.. note::
+
+   finalize() must be called after adding the final value and before
+ retrieving calculated statistics.
+.. seealso:: addValue()
+.. seealso:: calculate()
+.. seealso:: finalize()
+.. versionadded:: 2.16
+%End
 
-    /** Adds a single value to the statistics calculation. Calling this method
-     * allows values to be added to the calculation one at a time. For large
-     * quantities of values this may be more efficient then first adding all the
-     * values to a list and calling calculate().
-     * @param value variant containing to add. Non-numeric values are treated as null.
-     * @note call reset() before adding the first value using this method
-     * to clear the results from any previous calculations
-     * @note finalize() must be called after adding the final value and before
-     * retrieving calculated statistics.
-     * @see addValue()
-     * @see calculate()
-     * @see finalize()
-     * @note added in QGIS 2.16
-     */
-    void addVariant( const QVariant& value );
-
-    /** Must be called after adding all values with addValues() and before retrieving
-     * any calculated statistics.
-     * @see addValue()
-     * @see addVariant()
-     * @note added in QGIS 2.16
-     */
     void finalize();
+%Docstring
+ Must be called after adding all values with addValues() and before retrieving
+ any calculated statistics.
+.. seealso:: addValue()
+.. seealso:: addVariant()
+.. versionadded:: 2.16
+%End
 
-    /** Returns the value of a specified statistic
-     * @param stat statistic to return
-     * @returns calculated value of statistic
-     */
     double statistic( Statistic stat ) const;
```

Comment:
you might need to add `QgsStatisticalSummary`in front of `Statistic` (in the header)